### PR TITLE
appnexus bid adapter 5.0 - video params updates

### PR DESCRIPTION
**Pull request by jsnellbaker**
## Type of change

- [x] Feature

## Description of change
Related to https://github.com/prebid/Prebid.js/issues/6512

This change updates the appnexus bidder to read more from the `mediaTypes.video` params.  

It supports specific ORTB 2.5 video params that are closely aligned to video properties we currently support.  As discussed in the linked ticket, due to a number of unique values that are available/used by our ad-server but that are not supported by the ORTB spec - the `bids.params.video` are still read and would be preferred (if present) over the mediaType values.

### CODE REVIEW

**Comment by patmmccann** [comment]
Thanks! Could you also update https://github.com/prebid/Prebid.js/blob/appnexus5-video-params/modules/appnexusBidAdapter.md to demonstrate playback method isn't required to be in the params?

**Comment by jsnellbaker** [comment]
Made an update on the md file.

**Review by Fawke — APPROVED** [review]
LGTM

File: modules/appnexusBidAdapter.md [hunk 1]
```diff
@@ -89,7 +89,16 @@ var adUnits = [
...
        video: {
          playerSize: [[300, 250]],
-         context: 'outstream'
+         context: 'outstream',
+         // certain ORTB 2.5 video values can be read from the mediatypes object; below are examples of supported params
+         // to note - appnexus supports additional values for our system that are not part of the ORTB spec.  If you want
+         // to use these values, they will have to be declared in the bids[].params.video object instead using the appnexus syntax.
+         minduration: 1,
+         maxduration: 60,
+         skip: 1,   // 1 - true, 0 - false
```
Comment (patmmccann) [inline modules/appnexusBidAdapter.md:98]: In this example you have skip and playback method double set, is this intentional?

Comment (jsnellbaker) [reply modules/appnexusBidAdapter.md:98]: The idea is that you can set both - but the ones in the bid.params will taken precedence due to unique values we have to support.

I'll add another comment to better clarify this point.

**Review by patmmccann — APPROVED** [review]

**Pull request merged**